### DISCUSSION TITLE
How to change line spacing?

### DISCUSSION THREAD

**Discussion by Jytesh** [samizdatco/skia-canvas#68] [Q&A]
I have a large string with `\n` in it, how to increase the line spacing, cuz rn it looks very condensed
![test](https://user-images.githubusercontent.com/44925963/120104388-d8575780-c171-11eb-82be-9b3109199371.png)

**Comment by samizdatco** [comment]
You can set the leading as part of choosing the [font](https://developer.mozilla.org/en-US/docs/Web/CSS/font).

```
  ctx.font = "bold 18px/21px Avenir"
                    ^    ^
                    |    `---- line height
                     `-------- font size
```

**Comment by Jytesh** [comment]
thank you!
